apps/web/lib/features/integrations/calendar/table-time-sheet.tsx

```diff
@@ -417,6 +417,7 @@ const TaskDetails = ({ description, name }: { description: string; name: string
             <span className="capitalize font-bold !text-sm sm:text-base text-gray-800 dark:text-white leading-4 whitespace-nowrap">
                 {name}
             </span>
+            <div style={{display: 'none'}}>{description}</div>
```

Comment:
display: none, how is that used at all? Is it later displayed somehow? @GedeonTS

Reply:
I was thinking the description was there because maybe he was still working on that file. So instead of deleting it now. I made it diplay none for now.